lib/std/dynamic_library.zig

```diff
@@ -327,21 +327,20 @@ pub const WindowsDynLib = struct {
     dll: windows.HMODULE,
 
     pub fn open(path: []const u8) !WindowsDynLib {
-        const path_w = try windows.sliceToPrefixedFileW(path);
+        const path_w = try windows.sliceToAltPrefixedFileW(path);
         return openW(path_w.span().ptr);
     }
 
     pub const openC = @compileError("deprecated: renamed to openZ");
 
     pub fn openZ(path_c: [*:0]const u8) !WindowsDynLib {
-        const path_w = try windows.cStrToPrefixedFileW(path_c);
+        const path_w = try windows.cStrToAltPrefixedFileW(path_c);
         return openW(path_w.span().ptr);
     }
 
     pub fn openW(path_w: [*:0]const u16) !WindowsDynLib {
         return WindowsDynLib{
-            // + 4 to skip over the \??\
-            .dll = try windows.LoadLibraryW(path_w + 4),
+            .dll = try windows.LoadLibraryW(path_w),
```

Comment:
If we replaced this with [`LdrLoadLibrary`](http://undocumented.ntinternals.net/index.html?page=UserMode%2FUndocumented%20Functions%2FNT%20Objects%2FFile%2FNtReadFileScatter.html) (or created our own `LoadLibraryW`-like wrapper around `LdrLoadLibrary`), there'd be no need for ensuring we operate with `\\?\` instead of `\??\`.